handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/ServiceCombServerStats.java

```diff
@@ -35,15 +36,14 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(ServiceCombServerStats.class);
 
+  private final Object lock = new Object();
```

Comment:
lock is never read

Reply:
The `lock` is used in method `ensureWindow`. It does not get involved in this issue.